openreview/tools.py

```diff
@@ -15,6 +15,10 @@
 import urllib.parse as urlparse
 from tqdm import tqdm
 from concurrent.futures import ThreadPoolExecutor
+import json
+
+with open('duplicate_domains.json') as f:
```

Comment:
can you do a lazy load when `subdomains` is being called?

Reply:
okay, but that will make the `build_conflicts` call slow as it will load the JSON for every `subdomains` call.

Reply:
Done.